Sources/AngouriMath/Core/Entity/Omni/Entity.Matrix.cs

```diff
@@ -369,11 +350,19 @@ public IEnumerator<Entity> GetEnumerator()
             /// <summary>
             /// Adjugate form of a matrix
             /// </summary>
-            public Matrix Adjugate =>
+            public Matrix? Adjugate =>
                 adjugate.GetValue(static @this =>
-                    (Matrix)new Matrix(@this.InnerMatrix.Adjoint()).InnerSimplified,
+                    {
+                        if (!@this.IsSquare)
```

Comment:
Shouldn't this check be in GenericTensor?

Reply:
🦆

Reply:
🦆